src/rstar.jl

```diff
@@ -1,7 +1,8 @@
-
+import XGBoost
```

Comment:
Requires requires you to specify optional dependencies with a dot (the problem is that tests don't seem to cover this issue but users will get a warning otherwise):
```suggestion
using .XGBoost
```